src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -392,19 +392,23 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
   }
 
   renderTableHeader() {
-    const { renderTableHeader, isSelectable, isConfigurable, store } = this.props;
+    const { customizeTableRowProps, renderTableHeader, isSelectable, isConfigurable, store } = this.props;
 
     if (!renderTableHeader) {
       return;
     }
 
+    const enabledItems = customizeTableRowProps
+      ? this.items.filter(item => !customizeTableRowProps(item).disabled)
```

Comment:
Why do we need to use `customizeTableRowProps` here? I guess it's required only in items list rendering, but not in the header. By default `customizeTableRowProps` is `undefined`, maybe we should add default value?

```ts
const defaultProps: Partial<ItemListLayoutProps> = {
  customizeTableRowProps: (item) => ({} as TableRowProps),
  ...
```

And refactor also `item-list-layout.tsx:245` to:

```ts
 {...(customizeTableRowProps ? customizeTableRowProps(item) : {})} // before
 {...customizeTableRowProps(item)} // with default value
```

Wdyt?

```suggestion
      ? this.items.filter(item => !item.disabled)
```